metricbeat/module/system/_meta/kibana/7/dashboard/Metricbeat-Host-Services-overview.json

```diff
@@ -0,0 +1,779 @@
+            "gridData": {
+              "h": 11,
+              "i": "e3ec1fe3-a03a-466e-8bc2-69136e3e6302",
+              "w": 24,
+              "x": 0,
+              "y": 11
+            },
+            "panelIndex": "e3ec1fe3-a03a-466e-8bc2-69136e3e6302",
+            "panelRefName": "panel_4",
+            "version": "7.4.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 11,
+              "i": "cc928f52-22c9-402d-a7a1-4f32720a4290",
+              "w": 24,
+              "x": 24,
+              "y": 11
+            },
+            "panelIndex": "cc928f52-22c9-402d-a7a1-4f32720a4290",
+            "panelRefName": "panel_5",
+            "version": "7.4.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 14,
+              "i": "c83f16cd-286f-411b-bae9-bade176a8db2",
+              "w": 48,
+              "x": 0,
+              "y": 22
+            },
+            "panelIndex": "c83f16cd-286f-411b-bae9-bade176a8db2",
+            "panelRefName": "panel_6",
+            "version": "7.4.0"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat Services] Host Services Overview",
+        "version": 1
+      },
+      "id": "c431f410-f9ac-11e9-90e8-1fb18e796788",
+      "migrationVersion": {
+        "dashboard": "7.3.0"
+      },
+      "references": [
+        {
+          "id": "a30871f0-f98f-11e9-90e8-1fb18e796788",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "bb3a8720-f991-11e9-90e8-1fb18e796788",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "e6e639e0-f992-11e9-90e8-1fb18e796788",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "9c69cad0-f9b0-11e9-90e8-1fb18e796788",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "8c071e20-f999-11e9-90e8-1fb18e796788",
+          "name": "panel_4",
+          "type": "visualization"
+        },
+        {
+          "id": "4b254630-f998-11e9-90e8-1fb18e796788",
+          "name": "panel_5",
+          "type": "visualization"
+        },
+        {
+          "id": "d3f51850-f9b6-11e9-90e8-1fb18e796788",
+          "name": "panel_6",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-10-28T19:16:32.113Z",
+      "version": "WzE0MTAsMV0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.index",
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "Running Services [Metricbeat Services]",
```

Comment:
Please [customize the titles](https://github.com/elastic/beats/pull/13187#issuecomment-519911298) of the visualizations so the `[Metricbeat Services]` suffix doesn't appear.